python/openassetio/managerAPI/ManagerInterface.py

```diff
@@ -148,7 +148,7 @@ class ManagerInterface(_openassetio.managerAPI.ManagerInterface):
 
        @li @needsref identifier()
        @li @needsref displayName()
-       @li @ref info()
+       @li @needsref info()
```

Comment:
[fix] Can this be `@fqref{managerAPI::ManagerInterface::info} "info`"? (Sorry, for some reason I can't add suggestions).

Reply:
I just copied what was done for `identifier` and `displayName`, which I think were `@needsref`ed in the expectation that, once all these methods are migrated, stripping the `needs` bit should "just work".

I guess since we're now saying that there will be a relatively long-lived intermediate state, where only some methods are migrated, then it's worth making the cross-references work.

Reply:
You make an excellent point! lets just leave as `@needsref` then it is at least consistent...

Reply:
OK left as `@needsref` for now (though did fix a related issue in 4875942d)